src/plugins/intel_gpu/src/graph/fully_connected.cpp

```diff
@@ -187,7 +186,14 @@ kernel_impl_params fully_connected_inst::get_fake_aligned_params(kernel_impl_par
             return std::move(orig_impl_param);
         }
 
-        size_t fake_align_base = (orig_impl_param.dev_type == cldnn::device_type::integrated_gpu) ? 16 : 8;
+        size_t fake_align_base = 8;
```

Comment:
[random spot] Is this feature covered by existing test cases?

Reply:
Yes, some of tests cover new implementation as well, but I will add more tests in follow up PR